sentences/de/climate_HassClimateGetTemperature.yaml

```
@@ -6,5 +6,6 @@ intents:
           - "Wie hoch ist die Temperatur [<area>]"
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
-          - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | [von] <name>) <area>"
```

```suggestion
          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | (von | vom) <name>) [<area>]"
```
With this we can skip `Wie hoch ist die Temperatur (von | vom) <name>` (line 8)